apps/export/tests/test_apis.py

```diff
@@ -56,3 +56,19 @@ def test_trigger_api_with_export_permission(self):
         export = Export.objects.get(id=response.data['export_triggered'])
         self.assertTrue(export.pending)
         self.assertEqual(export.exported_by, self.user)
+
```

Comment:
We also need additional tests to make sure deleted exports are not available in list/retrieve/update APIs too. And non-owner users can't access other's exports.